webots_ros2_epuck/worlds/epuck_world.wbt

```diff
@@ -4,6 +4,7 @@ WorldInfo {
     "Simple e-puck simulation that can be controlled with ROS2."
   ]
   title "ROS2 simulation of the e-puck robot"
+  basicTimeStep 20
```

Comment:
Do we have to keep this `basicTimeStep`? I think it was set only for testing?

Reply:
If we want to have 50Hz publish rate then we have to put it.

Reply:
Ok fine for me too.